### DISCUSSION TITLE
Wired prefix in Neo-tree

### DISCUSSION THREAD

**Discussion by pawl1234** [LazyVim/LazyVim#4084] [Q&A]
Hi Community,

In neo-tree I have a prefix with every file in the tree. I guess its comming from my zsh configuration where I use a different tool instead of ls. Someone knows how I can fix this in lazyvim?

![image](https://github.com/user-attachments/assets/09bbf285-620a-4f5e-9ae9-7f766088a10b)

Thank you and greetings
Patrick

**Comment by dotfrag** [comment]
I'm not sure why ls would matter. I too use eza instead of ls. Do you know how to replicate this? When did it start happening? If you think it's related to zsh (I doubt it), you can try starting zsh with `zsh -f`, or `zsh --no-rcs`.

**Reply by folke** [reply]
Pretty sure this is a plugin doing that. And I don't think neo-tree uses ls. They most likely use libuv.

**Reply by dotfrag** [reply]
Yeah no way it uses ls.. if OP can provide some repro maybe it will make more sense.

**Reply by pawl1234** [reply]
Ok if its also the same if I'm using "zsh -f". I have a fresh and new lazyvim installation and until now it is all default. Also interesting might be, that this is not for all file extensions. Here for example for .ts its there but not for .tsx
![image](https://github.com/user-attachments/assets/7c98cd51-641e-463d-88c9-a02e6617dbe4)

**Reply by dotfrag** [reply]
I'm not sure how to help if I can't replicate it. Was this always happening or did it start happening recently? Are your fonts up to date? Do you get this when using `repro.lua` from bug reports, or LazyVim starter?
